README.md

```diff
@@ -1058,59 +1077,59 @@ This also applies for typehints in PhpDoc of properties.
 Examples:
 ```php
 <?php
- 
+
 declare(strict_types=1);
- 
+
 class SomeClass
 {
     /**
-     * @var int|null    –> this must include `|null`, as `$id` can be uninitialized 
+     * @var int|null    –> this must include `|null`, as `$id` can be uninitialized
      */
     private $id;
- 
-    public function getId(): ?int   // return value here must be nullable 
+
+    public function getId(): ?int   // return value here must be nullable
     {
         return $this->id;
     }
- 
+
     /**
      * @return int|null
      */
-    public function getIdBefore71() 
+    public function getIdBefore71()
     {
         return $this->id;
     }
- 
+
     public function setId(int $id): self    // we can use non-nullable type for setter, though
     {
         $this->id = $id;
-         
+
         return $this;
     }
 }
 ```
 
 ```php
 <?php
- 
+
 declare(strict_types=1);
- 
+
 class SomeClass
 {
     /**
-     * @var Child|null 
+     * @var Child|null
      */
     private $child;
```

Comment:
why docBlock is the correct sample if we are asking to use type hints? It's not a function, which may throw an exception, so no doc block is required, while getChild() seems requiring docBlock